<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_info_genere</class>
 <widget class="QDialog" name="Dialog_info_genere">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>204</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Informazioni Genere</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_genere">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>331</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Horror</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabFilm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>411</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>#film: -        #visti: -</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>134</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Film di questo genere:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
